axon/controller/axon_rpyc_controller.py

```diff
@@ -10,13 +10,14 @@
 import rpyc
 from rpyc.utils.server import ThreadPoolServer
 
-from axon.common import config as conf
```

Comment:
If i remember correctly, this conf import was put on top because it setups the logging first. In future we will change that but for time being we need it.